.gitignore

```diff
@@ -46,3 +46,6 @@ examples/libraries/restricted-extras-qscintilla/restricted-extras-qscintilla
 # android temporary build files
 android-build
 deployment-settings.json
+
+*.docker-buildstamp
+compile_flags.txt
```

Comment:
What is compile_flags.txt?

Reply:
ah, that's something [`clangd` uses](https://clangd.llvm.org/installation#compile_flagstxt) to provide (more) accurate intellisense - can remove or leave a comment, whichever you prefer.

Reply:
If it's for something well-known like clangd LSP, that's fine as-is. :ok_hand: